Client/src/bkr/client/__init__.py

```diff
@@ -81,6 +81,8 @@ def set_hub(self, username=None, password=None, **kwargs):
         if kwargs.get('insecure'):
             self.conf['SSL_VERIFY'] = False
         proxy_user = kwargs.get('proxy_user')
+        if 'USERNAME' in self.conf:
```

Comment:
This seems to override the value if `username` is passed in on L78.

Maybe it should be:

    if username is None and if 'USERNAME' in self.conf:

Reply:
Thank you! That's a fair criticism. I've updated and force-pushed this change, and it is ready for review again.